code/procs/scorestats.dm

```diff
@@ -352,6 +355,33 @@ var/datum/score_tracker/score_tracker
 
 		return jointext(., "")
 
+	//I feel like I may be reinventing the wheel here, but here's a recursive wheel
+	//recursively finds all items of type in container
+	proc/find_items_in(obj/container, type)
+		var/list/found = new /list(0)
+		for (var/obj/item/item in container.contents)
+			if (istype(item, type))
+				found += item
+			found += find_items_in(item, type)
+
+		return found
+
+	proc/get_inspector_report()
+		var/report = ""
+		//this might be a really stupid way of finding all players of a specific job, please tell me if it is
+		for (var/mob/living/carbon/human/player in mobs)
+			if (player.job != "Inspector")
+				continue
+			report += "<B>Inspector [player.real_name]'s report</B><BR><HR>"
+			//find all clipboards they have on them (probably just one, but may as well check)
+			var/list/clipboards = find_items_in(player, /obj/item/clipboard)
+			for (var/obj/item/clipboard/clipboard as anything in clipboards)
```

Comment:
If player has multiple clipboards, multiple reports will be shown. I would suggest a "break" in the for loop after it's found and done 1 clipboard.

Reply:
I had it written like that originally, but thought that a player randomly picking up an empty clipboard isn't too uncommon an event and then it's a 50/50 chance whether their report actually gets shown. It shouldn't show up as a separate report exactly, just add on to the end of the current one.

Reply:
Also empty papers/clipboards won't show up on the endscreen